src/NServiceBus.Core/Routing/RoutingFeature.cs

```diff
@@ -87,14 +103,17 @@ protected internal override void Setup(FeatureConfigurationContext context)
             }
         }
 
-        static void ImportMessageEndpointMappings(FeatureConfigurationContext context, MessageEndpointMappingCollection legacyRoutingConfig, TransportInfrastructure transportInfrastructure, Publishers publishers, UnicastRoutingTable unicastRoutingTable)
+        static Type[] GetKnownMessageTypes(FeatureConfigurationContext context)
```

Comment:
I think I'm touching some of that code too in a follow up PR to fix polymorphism. Are you ok with setting #3948 back a bit and wait until we finished #3911 ?